src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -88,8 +88,7 @@ public boolean isSameContact(Contact otherPerson) {
 
         return otherPerson != null
```

Comment:
I think this should be changed from otherPerson to otherContact? likewise for the JavaDocs of this method.

Reply:
Thanks for the note, it has been corrected.